MapboxDirections/MBLane.swift

```diff
@@ -56,27 +89,48 @@ public enum LaneIndicationType: Int, CustomStringConvertible {
             return "slight left"
         case .SlightRight:
             return "slight right"
-        case .Straight:
+        case .StraightAhead:
             return "straight"
         case .Uturn:
             return "uturn"
+        case .None:
+            return "None"
         }
     }
 }
 
+@objc(MBLane)
 public class Lane: NSObject {
+    
+    /**
+     A boolean indicating whether the lane is a valid choice in the current maneuver.
+    */
     public var validTurn: Bool
-    public var indications: [LaneIndicationType]
     
-    internal init(validTurn: Bool, indications: [LaneIndicationType]) {
+    /**
+     An indication (e.g. marking on the road) specifying the turn lane.
+     
+     A road can have multiple indications (e.g. an arrow pointing straight and left). The indications are given in an array, each containing one of the following types. Further indications might be added on without an API version change.
+    */
+    public let indications: [LaneIndication]
+    
+    
+    /**
+     Objectice-c helper function for indication
```

Comment:
`indications` won’t even appear to an Objective-C developer. You’ll have to document this property as if `indications` doesn’t even exist.